data/net/minecraft/world/entity/Entity.mapping

```diff
@@ -105,8 +108,15 @@ CLASS net/minecraft/world/entity/Entity
 		COMMENT Plays the {@link #getSplashSound() splash sound}, and the {@link ParticleType#WATER_BUBBLE} and {@link ParticleType#WATER_SPLASH} particles.
 	METHOD ejectPassengers ()V
 		COMMENT Dismounts all entities riding this entity from this entity.
+	METHOD equals (Ljava/lang/Object;)Z
+		ARG 1 other
 	METHOD fillCrashReportCategory (Lnet/minecraft/CrashReportCategory;)V
 		ARG 1 category
+	METHOD fillIndirectPassengers (ZLjava/util/Set;)V
+		ARG 1 checkPlayer
```

Comment:
Why `checkPlayer`? From how it's used -- to skip any (server)player entities from being added to the set passed to the method -- I'd say a more appropriate name would be along the lines of `skipPlayer`.

Reply:
I think it should be left unchanged, or renamed `onlyPlayer`, I think it's for checking if It should only include `ServerPlayer`, or every entities

Reply:
I think it should be left unchanged, or renamed `onlyPlayer`, I think it's for checking if It should only include `ServerPlayer`, or every entities

Reply:
Hmmm, it seems I've misread the code. If we look at the loop as seen via Enigma:
```java
if (!onlyPlayers || ServerPlayer.class.isAssignableFrom(passenger.getClass())) {
    fillTo.add(passenger);
}
passenger.fillIndirectPassengers(skipPlayer, fillTo)
```
So the passenger entity is added to the set if either `onlyPlayers` is `false` or the passenger entity is an instance of `ServerPlayer`. So yeah, disregard my comment, but I would request that, as you suggest, it be renamed to `onlyPlayers` instead of `checkPlayer` (as the new name is more intuitively understandable as adding only players to the set).

Reply:
I have changed parameter name to `onlyPlayer`